### ISSUE TITLE
modules location suggestion

### ISSUE THREAD

**Issue by luizbills** [joelnet/MojiScript#110]
I think `IfElse`, `IfError`, `cond`, `unless`, `when` location should be `logic` instead of `core`.

**Comment by joelnet** [comment]
This is a good recommendation. logic is new and I should have moved those over when it was created.

**Comment by luizbills** [comment]
Or we can move **all** scripts to root, so nobody needs to decorate locations. :smile: 

```js
import pipe from 'mojiscript/pipe'
import run from 'mojiscript/run'
import log from'mojiscript/IfElse'
// or all with a same location
import pipe from 'mojiscript/lib/pipe'
import run from 'mojiscript/lib/run'
import log from'mojiscript/lib/IfElse'
```

**Comment by joelnet** [comment]
That is another option. I first started to organize them into categories. But the problem is some things are hard to put into one category. There's also the problem of moving something from one category to another. Also I hope that there isn't a monster library created. It would be nice to keep MojiScript to a limit set of core functions. Then supplement the library with other libs like Ramda or Sanctuary.

What are your thoughts on that?

**Comment by luizbills** [comment]
I agree. I prefer a small and lightweight core too. For that reason, I think it would be better to put everything (of mojiscript) in a unique category.

It would also be nice to recommend a supplementary library (+1 for sanctuary) and show implementation examples (to help newbies, like me).

The mojiscript's core then will just be something to run asynchronous and synchronous functional code together? That is the idea?

**Comment by joelnet** [comment]
I looked into flattening the structure, but it ended up looking too disorganized. the combinators were sprinkled all over, the console files were just labeled `error` and `log`, which seemed similar to `ifError` and `string/append` is just called `append`, but it's not clear that it's just for strings.

It's gonna take more work to flatten the structure. So for now I just moved `cond`, `ifElse`, `ifError`, `unless` and `when` into `logic`.
